### PR TITLE
Fixes undefined variable "profile"

### PR DESCRIPTION
**Pull request by Juri-Salminen**
The load()-function in +page.js returns the "profile"-object referenced as "data" in +layout.svelte, hence it should be referenced as "data.profile" instead of "profile".

### CODE REVIEW

File: src/routes/profile/%40[user]/+layout.svelte [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
 	let current_token;
 	async function toggle_following() {
 		const token = (current_token = {});
 
-		const { following, username } = profile;
+		const { following, username } = data.profile;
 
 		// optimistic UI
-		profile.following = !profile.following;
+      data.profile.following = !data.profile.following;
```
Comment (benmccann) [inline src/routes/profile/%40[user]/+layout.svelte:20]: can you change the spaces back to tabs on this line and the one below?

**Comment by benmccann** [comment]
Did you test this fix to confirm that follow toggling works? It's obviously less broken than the existing code, but I'm not entirely sure it's valid to update `data.profile` in that manner

File: src/routes/profile/%40[user]/+layout.svelte (hunk 1 above)
Comment (Juri-Salminen) [reply src/routes/profile/%40[user]/+layout.svelte]: I hope it's fixed now :)

File: src/routes/profile/%40[user]/+layout.svelte [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
 	let current_token;
 	async function toggle_following() {
 		const token = (current_token = {});
 
-		const { following, username } = profile;
+		const { following, username } = data.profile;
 
 		// optimistic UI
-		profile.following = !profile.following;
+      	data.profile.following = !data.profile.following;
```
Comment (benmccann) [inline src/routes/profile/%40[user]/+layout.svelte:20]: this line is still not indented correctly

**Pull request closed without merging**